### DISCUSSION TITLE
Change the line length to the Black default of 88

### DISCUSSION THREAD

**Discussion by cpcloud** [ibis-project/ibis#4515] [Polls]
If you're against changing the line length to the Black default of 88, then please give a short reason why.

The idea here is to reduce vertical whitespace, use defaults from black, and have a line length that is more optimized for readabilty.

**Comment by saulpw** [comment]
I'd also suggest and vote for a line length of 92, if it's being changed.

**Comment by kszucs** [comment]
I have a slight preference towards 79, but I'd probably not against 88. Could you perhaps put up a PR with line length 88 just for comparison purposes?

**Reply by cpcloud** [reply]
Sure. Moving to 88 shaves around 2.2k lines of the current codebase for reference.

**Reply by cpcloud** [reply]
It's actually close to 3k lines https://github.com/ibis-project/ibis/pull/4517.

**Comment by cpcloud** [comment]
This is done.
